src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,61 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  min-width: $iui-3xl * 2;
+  width: $iui-3xl * 4;
+  max-width: 50vw;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 0 $iui-sm;
+  overflow-x: hidden;
+  overflow-y: auto;
+  overflow-y: overlay;
+  resize: horizontal;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-right: 1px solid t(iui-color-background-5);
+  }
+
+  @include iui-transition-group;
+
+  &-header {
+    height: $iui-baseline * 5;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-label {
+      @include iui-text(title);
+      display: flex;
+      align-items: center;
+      overflow: hidden;
+
+      .iui-button {
+        flex-shrink: 0;
+      }
+
+      > * {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+
+      &-actions {
+        flex-shrink: 0;
+        margin-left: $iui-s;
+      }
+    }
+  }
+
+  &-body {
```

Comment:
Is this class really needed? Maybe just add `padding` to all directions in `iui-side-navigation-submenu`.

Reply:
If I add `padding-top` to `.iui-side-navigation-submenu` I am unable to align the `.iui-side-navigation-submenu-header` with the first button in the side navigation.

I've removed the `.iui-side-navigation-submenu-body` class and instead added a bottom padding to `.iui-side-navigation-submenu`.

Only thing we should be cautious of is if a user does _not_ use the `.iui-side-navigation-submenu-header` class we will not have any top padding present.  This isn't standard, so hopefully shouldn't be an issue.

Reply:
Do you still need the extra div?
![image](https://user-images.githubusercontent.com/9084735/134973469-b437b702-9cf5-47ec-b45d-2bbebfaff124.png)

Reply:
Seems like not needed, removed.